Resources/Prototypes/_White/Aspects/Aspects.yml

```
@@ -88,3 +88,104 @@
     startAudio:
       path: /Audio/_White/Aspects/accent.ogg
   - type: ImmersiveAspect
+
+- type: entity
+  id: RulonOboevAspect
+  parent: BaseGameRule
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Aspect
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)